apps/api/src/app/workflows-v2/usecases/build-payload-schema/build-payload-schema.usecase.ts

```diff
@@ -16,26 +15,13 @@ export class BuildPayloadSchema {
 
   @InstrumentUsecase()
   async execute(command: BuildPayloadSchemaCommand): Promise<JSONSchemaDto> {
-    const controlValues = await this.buildControlValues(command);
+    const controlValues = await this.getControlValues(command);
+    const sanitizedControlValues = await this.sanitizeControlValues(controlValues);
```

Comment:
i am not sure if the naming is correct here we do not sanitizeControlValues we create templateVariableNames and the function is returning not control value but string[] of variables

Reply:
Fixed